addon/state-for.js

```diff
@@ -61,16 +61,16 @@ export default function stateFor(stateName, propertyName) {
  * on the instance if desired.
  */
 function createStateFor(context, stateName, container) {
-  let defaultState  = {};
-  let containerName = `state:${stateName}`;
-  let StateFactory  = container.lookupFactory(containerName);
+  let defaultState    = {};
+  const containerName = `state:${stateName}`;
+  let StateFactory    = container.lookupFactory(containerName);
 
   if (!StateFactory) {
-    throw new TypeError(`Unknown StateFactory: ${containerName}`);
+    StateFactory = {};
```

Comment:
What should the default state be? a blank POJO or an ember object? @stefanpenner